Web/Controllers/PaymentV2Controller.cs

```diff
@@ -119,6 +119,10 @@ public async Task<IActionResult> Payments()
         [FromRoute] string paymentAuthorizationId)
     {
         if (_environment.IsProduction()) return NotFound();
+
+        return NotFound();
+        
+        /*
         try
```

Comment:
should we keep it commented-out? should we just remove it?

Reply:
I think I just forgot about it 😅 
I'll add the feature back - thanks!